django_project/minisass_frontend/src/components/Map/Map.tsx

```diff
@@ -77,7 +77,12 @@ export const Map = forwardRef((props: Interface, ref) => {
             const urlTile = new URL(styles.sources[minisassObservationId].tiles[0])
             const currUrl = new URL(window.location)
             currUrl.pathname = urlTile.pathname
-            styles.sources[minisassObservationId].tiles[0] = decodeURIComponent(currUrl.href)
+            // styles.sources[minisassObservationId].tiles[0] = decodeURIComponent(currUrl.href)
+            styles.sources[minisassObservationId].tiles[0] = 'http://192.168.1.7:7800/tiles/public.minisass_observations/{z}/{x}/{y}.pbf#/map'
```

Comment:
@zamuzakki why are you using this ip address?

Reply:
@dimasciput I manually update the tile URL on local since it's using production tiles. I have reverted it